apps/prairielearn/elements/pl-number-input/pl-number-input.mustache

```diff
@@ -12,26 +12,53 @@
 {{#inline}}<span class="form-inline d-inline-block ml-2">{{/inline}}
     <span id="pl-number-input-{{uuid}}" class="input-group pl-number-input">
         {{#label}}
+        <label for="pl-number-input-textfield-{{uuid}}" hidden>{{accessibility_description}}</label>
         <span class="input-group-prepend">
-            <span class="input-group-text">{{{label}}}</span>
+            <span class="input-group-text" id="pl-number-input-{{uuid}}-label">{{{label}}}</span>
         </span>
         {{/label}}
-        <input name="{{name}}" autocomplete="off" type="text" class="form-control {{^display_append_span}}rounded-right{{/display_append_span}} pl-number-input-input" size="{{size}}" {{^editable}}disabled{{/editable}} {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}} aria-describedby="basic-addon1" placeholder="{{shortinfo}}" />
```

Comment:
Now that the rounded-right logic is removed here, I think we can remove the `display_append_span` key from the dict passed into this mustache template (I don't think that key is used anywhere else).

Reply:
I checked, and it looks like I have removed the `display_append_span` keys per [9743a97](https://github.com/PrairieLearn/PrairieLearn/pull/7934/commits/9743a9740eb46720989ebb29be003e26577beec8), when the rounded-right logic was removed. I think I was wise to the fact that the keys have no use aside from serving the function of `rounded-righ`, and so the keys were removed then.

Reply:
It looks like the key is still being set @Frozemint ? Check line 275: https://github.com/PrairieLearn/PrairieLearn/blob/d7f4c02c81343c02646b3e6e5e2a423a7b2b0e69/apps/prairielearn/elements/pl-number-input/pl-number-input.py#L275

Reply:
Good catch 💀